Rebekahperillo patch 2

rebekahperillo opened the pull request:
Full update of training content for travel A/OPCs.

weiwang-gsa commented on training-front-end/src/content/training_travel_pc/lesson09.mdx:
```diff
@@ -0,0 +1,25 @@
+## Where am I able to access the GSA SmartPay Master Contract?
+Understanding the terms and conditions of the GSA SmartPay Master Contract is important to performing your role as an A/OPC. The terms and conditions of the Master Contract identify specific contractual requirements that the GSA SmartPay program has with the contracting banks.
+
+The [GSA SmartPay Master Contract](https://smartpay.gsa.gov/content/gsa-smartpay-master-contract) can be viewed on the [GSA SmartPay website](https://smartpay.gsa.gov/). Download a copy and review relevant clauses and sections that pertain to the GSA SmartPay Travel program, as well as to the GSA SmartPay program in general.
```
is this link from legacy site? may need to be updated with link to current site structure.

mark-meyer replied:
I think Jeff's work on redirects will allow this link to work when we switch over since he added the redirect:

```
"/content/gsa-smartpay-master-contract": "/about/master-contract"
```

but this is a good note that we should plan on searching through the content here for links going to the Program site and update them when the domain switched so we link to the correct place.

weiwang-gsa commented on training-front-end/src/content/training_travel_pc/lesson08.mdx:
```diff
@@ -1,25 +1,106 @@
+- GSA developed and hosts an online training course for travel card/account holders that discusses the proper use of travel card/account.
+- The annual GSA SmartPay Training Forum for A/OPCs provides training on the bank’s EAS, best practices and program management.
+- Free online resources from the [GSA SmartPay website](https://smartpay.gsa.gov/) to assist travel A/OPCs in detecting and preventing misuse and fraud.
+- Printable resources such as [Helpful Hints for Travel Account Use](https://smartpay.gsa.gov/sites/default/files/downloads/19-00248_19%20SmartPay%20Travel%20Helpful%20Hints_508C.pdf) is a card-sized brochure that provides information on the travel card/account. This brochure can be ordered online and can be passed out to card/account holders when they receive their travel cards/accounts.
```
the link to Helpful hints for Travel Account Use may need updates, this link looks like from the legacy sites, maybe need to be updated to use correct link to current program sites.

weiwang-gsa reviewed (approved):
Looks great to me! there are some links looks like still pointing to legacy site, may need a note for a  final update later on.

rebekahperillo commented:
Yes, we will have to clean those up. I assume we just wait on that for now?

Thank you, Wei!

Rebekah Knouse Perillo (she/her)

Lead Business Management Specialist

Center for Charge Card Management
Office of Professional Services and Human Capital Categories
Federal Acquisition Service, U.S. General Services Administration

301-938-5257

***@***.***


Follow the GSA SmartPay program on social media...Tweet with us on Twitter
<https://twitter.com/GSASmartPayNews> and friend us on Facebook
<https://www.facebook.com/GSASmartPayNews/>!
[image: image.png]


On Fri, Aug 25, 2023 at 1:11 PM Wei Wang ***@***.***> wrote:

> ***@***.**** approved this pull request.
>
> Looks great to me! there are some links looks like still pointing to
> legacy site, may need a note for a final update later on.
> ------------------------------
>
> In training-front-end/src/content/training_travel_pc/lesson09.mdx
> <https://github.com/GSA/smartpay-training/pull/308#discussion_r1305923811>
> :
>
> > +
> +### Citibank
> +- 800-790-7206 (within United States)
> +- 904-954-7850 (collect calls from outside United States)
> +- [Citibank Online Account Access](https://home.cards.citidirect.com/CommercialCard/Cards.html) (self register, commercial site).
> +
> +### U.S. Bank
> +- 888-994-6722 (within United States)
> +- Note: To speak with a live person, do not press any buttons.
> +- 701-461-2232 (collect calls from outside United States)
> +- [U.S. Bank Online Account Access](https://access.usbank.com/cpsApp1/index.jsp) (self register, commercial site).
> +
> +## Where am I able to access the GSA SmartPay Master Contract?
> +Understanding the terms and conditions of the GSA SmartPay Master Contract is important to performing your role as an A/OPC. The terms and conditions of the Master Contract identify specific contractual requirements that the GSA SmartPay program has with the contracting banks.
> +
> +The [GSA SmartPay Master Contract](https://smartpay.gsa.gov/content/gsa-smartpay-master-contract) can be viewed on the [GSA SmartPay website](https://smartpay.gsa.gov/). Download a copy and review relevant clauses and sections that pertain to the GSA SmartPay Travel program, as well as to the GSA SmartPay program in general.
>
> is this link from legacy site? may need to be updated with link to current
> site structure.
> ------------------------------
>
> In training-front-end/src/content/training_travel_pc/lesson08.mdx
> <https://github.com/GSA/smartpay-training/pull/308#discussion_r1305927337>
> :
>
> > +### Account Deactivation
> +- In those instances when the travel card/account is not needed on a continuous basis, deactivation of the account may serve as a deterrent to fraud and/or misuse.
> +- You may deactivate the card/account when a card/account holder is not using or is not planning to use the travel card/account.
> +- By understanding the card/account holder's need and use of the account, you can work with the card/account holder to establish deactivation guidelines.
> +- Deactivation and reactivation can be completed through the bank's EAS or by calling the bank's customer service phone number.
> +
> +### Guides
> +- The banks have developed written guides for A/OPCs and card/account holders.
> +- A/OPC Guide: This guide addresses issues of concern to the A/OPC, including responsibilities of program participants, account setup and maintenance, account suspension/cancellation, disputes, reports and invoicing procedures. The guide is available from the banks in hard copy and/or electronically.
> +- Card/Account Holder Guide: This guide addresses authorized uses of the travel card/account, disputes, and billing.
> +
> +### GSA Provided Resources
> +- GSA developed and hosts an online training course for travel card/account holders that discusses the proper use of travel card/account.
> +- The annual GSA SmartPay Training Forum for A/OPCs provides training on the bank’s EAS, best practices and program management.
> +- Free online resources from the [GSA SmartPay website](https://smartpay.gsa.gov/) to assist travel A/OPCs in detecting and preventing misuse and fraud.
> +- Printable resources such as [Helpful Hints for Travel Account Use](https://smartpay.gsa.gov/sites/default/files/downloads/19-00248_19%20SmartPay%20Travel%20Helpful%20Hints_508C.pdf) is a card-sized brochure that provides information on the travel card/account. This brochure can be ordered online and can be passed out to card/account holders when they receive their travel cards/accounts.
>
> the link to Helpful hints for Travel Account Use may need updates, this
> link looks like from the legacy sites, maybe need to be updated to use
> correct link to current program sites.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GSA/smartpay-training/pull/308#pullrequestreview-1596156644>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7Z6HEXV6AEHYM3SZBTEP2TXXDMCFANCNFSM6AAAAAA36YKKVY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

mark-meyer reviewed (approved):
This looks great @rebekahperillo!

rebekahperillo commented:
Thank you, Mark! And if the redirects don't work, we'll fix them post
launch.

Rebekah Knouse Perillo (she/her)

Lead Business Management Specialist

Center for Charge Card Management
Office of Professional Services and Human Capital Categories
Federal Acquisition Service, U.S. General Services Administration

301-938-5257

***@***.***


Follow the GSA SmartPay program on social media...Tweet with us on Twitter
<https://twitter.com/GSASmartPayNews> and friend us on Facebook
<https://www.facebook.com/GSASmartPayNews/>!
[image: image.png]


On Fri, Aug 25, 2023 at 2:16 PM Mark Meyer ***@***.***> wrote:

> ***@***.**** approved this pull request.
>
> This looks great @rebekahperillo <https://github.com/rebekahperillo>!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GSA/smartpay-training/pull/308#pullrequestreview-1596278705>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7Z6HETYLXBOGOBV3BZZVT3XXDTXBANCNFSM6AAAAAA36YKKVY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
